### ISSUE TITLE
[2.0] Jack-in to CLI/deps projects does not seem to work

### ISSUE THREAD

**Issue by PEZ** [BetterThanTomorrow/calva#157]
When jacking in to CLJ/CLI/deps.edn projects the process never completes to a successful jack-in. I can sometimes see the repl prompt being presented in the ”Calva Jack-in” output channel, but it is as if the jack-in code is waiting for something more to happen, which never happens.

Not sure if this is with all CLI projects, but I have hunted down a few and haven't found any that works,

**Comment by xavi** [comment]
I guess the issue corresponds to the not yet released version 2.0 of Calva, but FWIW, here's my experience with a CLI/deps project and Calva 1.3.66.

I have a `hello-calva` folder with a `deps.edn` file like...

```
{:paths ["src"]
 :aliases {:nrepl {:extra-deps {cider/cider-nrepl {:mvn/version "0.21.1"}}
                   :main-opts ["-m" "nrepl.cmdline"]}}}
```

and a `src/hello.clj` file like...

```
(ns hello)

(println "hello Calva!")
```

I open `hello-calva` folder in VSCode and, in Settings, set "Calva: ConnectREPLCommand" to

```
clj -Sdeps '{:deps {reply {:mvn/version "0.4.3"}}}' -m reply.main --color --attach
```

From the VSCode integrated Terminal I execute

```
clj -A:nrepl
```

I click on the nREPL in the lower left of the editor window and the dialog is automatically filled in with 'localhost:' followed by the port opened by nREPL. I press enter.

I open hello.clj and use Shift+Cmd+P to invoke the command palette and select "Calva: Evaluate current file".

In VSCode Terminal's OUTPUT tab, with "Calva says" selected in the dropdown, I see "hello Calva!".

If I click the TERMINAL tab, and select "Clojure REPL" in the dropdown, there's a

```user=> WARNING: cat already refers to: #'clojure.core/cat in namespace: net.cgrand.parsley.fold, being replaced by: #'net.cgrand.parsley.fold/cat```

followed by a cursor, but no prompt. However, if I press enter, I get the prompt...

```user=>```

If I place the cursor after the last parenthesis in hello.clj's `println` and invoke the command palete (Shift+Cmd+P) and select "Calva: Evaluate selection or current form", in the Terminal's OUTPUT tab, with "Calva says" selected in the dropdown, I see this being printed

```
out: hello Calva!
=> nil
```

That's fine, but...

  "View docstrings on hover"

doesn't work. Is there a known way to make it work?

**Comment by PEZ** [comment]
This now seem to work. :smile:
